trademarks.md

```diff
@@ -89,6 +89,14 @@ In those cases, you may use the word "Jupyter" or the unaltered logos to indicat
 without our prior approval.
 This is true both for non-commercial and commercial uses.
 
+Use of stills, videos and screenshots of the Jupyter applications (included but
+not limited to: notebook, jupyterLab, jupyterHub, nbconvert, IPython) as well as
+Jupyter Websites are allowed. Any stills, videos, and screenshots of a specific
```

Comment:
Maybe it would actually be worth saying "a specific notebook document, or any other file displayed by the application" (since e.g. if it's a screenshot of a Python script, we would still want it to be subject to that code's license)